adam-core/src/main/scala/org/bdgenomics/adam/rdd/contig/NucleotideContigFragmentRDD.scala

```diff
@@ -24,22 +24,16 @@ import org.apache.spark.api.java.JavaRDD
 import org.apache.spark.rdd.RDD
 import org.apache.spark.sql.{ Dataset, SQLContext }
 import org.bdgenomics.adam.converters.FragmentConverter
-import org.bdgenomics.adam.models.{
-  ReferenceRegion,
-  ReferenceRegionSerializer,
-  SequenceRecord,
-  SequenceDictionary
-}
+import org.bdgenomics.adam.models.{ ReferenceRegion, ReferenceRegionSerializer, SequenceDictionary, SequenceRecord }
```

Comment:
Nit: Can you revert the multiline to single line import change? We break lines at 80 chars.